### PR TITLE
Activity bucket

### PR DESCRIPTION
**Pull request by AnthonyLaw**
### Added
- Latest Activity bucket for Account Detail. #217 
- dynamic disable clickable link. #213 

### modified 
- Move Multisig Info data in to Account Detail Table.
- formatted LinkedAccount from Public Key to Plain Address.
- clickable redirect to account detail For LinkedAccount and Remote Account in `LinkAccount` Transaction
- Rename `store/Account`'s `getter` and `mutations`

### Remove
- Multisig Info `Card` in Account Detail

### Minor Fix
- `Show Detail` in Aggregate able to pop up the modal.
- Retrieve networkType via `http.networkType` rather than hard coded.

### CODE REVIEW

File: src/views/AccountDetail.vue [hunk 1]
```diff
@@ -1,104 +1,83 @@
...
+            :value="selectedTransactionType"
+            :options="transactionTypes"
+            @change="changeTransactionType"
+          />
+        </template>
+
+        <template #body>
+          <TableListView :data="filteredTransactionList" />
+        </template>
+      </Card>
```
Comment (Alexhuszagh) [inline src/views/AccountDetail.vue]: All the other Vue templates use 4-space indentation now, can you undo the formatting changes to this file? See [Card.vue](https://github.com/nemfoundation/nem2-explorer/blob/master/src/components/containers/Card.vue) for a guide.

File: src/components/tables/TableListView.vue [hunk 2]
```diff
@@ -66,19 +66,19 @@ export default {
   components: { Modal, AggregateTransaction },
   props: {
     data: {
-      type: Array,
-      required: true,
+      type: [Array, Object],
```
[the remark this answers is not in the document]
Comment (Alexhuszagh) [reply src/components/tables/TableListView.vue]: I don't see any reason for this to be an object.

**Comment by Alexhuszagh** [comment]
@AnthonyLaw I think I'm going to change the lint rules prior to committing this.

File: src/components/tables/TableListView.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (AnthonyLaw) [reply src/components/tables/TableListView.vue]: I was getting the error message from the component.

Screenshot 2019-11-14 at 2 00 50 PM

So i believe it cause by `AggregateTransaction` Component.

**Pull request merged**